RecoPixelVertexing/PixelVertexFinding/test/VertexFinder_t.h

```diff
@@ -0,0 +1,347 @@
+#include <cmath>
+#include <cstdint>
+#include <iostream>
+#include <random>
+#include <vector>
+
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/requireDevices.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/launch.h"
+#ifdef USE_DBSCAN
+#include "../plugins/gpuClusterTracksDBSCAN.h"
+#define CLUSTERIZE gpuVertexFinder::clusterTracksDBSCAN
+#elif USE_ITERATIVE
+#include "../plugins/gpuClusterTracksIterative.h"
+#define CLUSTERIZE gpuVertexFinder::clusterTracksIterative
+#else
+#include "../plugins/gpuClusterTracksByDensity.h"
+#define CLUSTERIZE gpuVertexFinder::clusterTracksByDensityKernel
+#endif
+#include "../plugins/gpuFitVertices.h"
```

Comment:
I can't recall if we have an exception for allowing relative paths.
@cms-sw/core-l2 are there any concerns about this? Perhaps a footnote in the rules may be necessary